integration-tests/src/test/groovy/com/okta/sdk/tests/it/GroupsIT.groovy

```diff
@@ -153,10 +153,11 @@ class GroupsIT implements CrudTestSupport {
 
         // 2. Add user to the group and validate user present in group
         user.addToGroup(group.getId())
-        assertUserInGroup(user, group)
+        // try upto 5 times with a delay of 2000ms after each failed attempt
```

Comment:
Ran Travis build couple of times and verified it working.

Reply:
@arvindkrishnakumar-okta That's great. I'm running some loops to try local runs. How often did you run it on Travis?

Reply:
Approx 5 - 7 times and tests passed everytime.